charts/kubewarden-defaults/Chart.yaml

```
@@ -22,9 +22,9 @@ keywords:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.6.0
+version: 3.6.1
 # This is the version of Kubewarden stack
-appVersion: v1.28.0
+appVersion: v1.28.1
```

You should not update the `appVersion`. This version is bumped only when we have a whole Kubewarden stack update. In this case, you should bump the `version` only.

---

Jose is right about that

```suggestion
appVersion: v1.28.0
```